### ISSUE TITLE
Plot copy number integer with segment

### ISSUE THREAD

**Issue by jnaccara** [etal/cnvkit#682]
Is there a way to display copy number integer when using cnvkit.py scatter? I just started using cnvkit this week and cannot find the option. Ideally, the copy number integr would be displayed on the segment in the scatterplot.

**Comment by ouslalu** [comment]
The .cns file from which the scatter diagram is plotted does not have the copy number integer information. You might want to check the .call.cns file.

**Comment by etal** [comment]
No, it's not an option yet. CNVkit needs another step to estimate integer copy numbers; initially it only estimates read depth log2-ratios. 
https://cnvkit.readthedocs.io/en/stable/calling.html
